kolibri/core/assets/src/views/buttons-and-links/k-external-link.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+
+  <a :class="buttonClasses" :href="href">
+    <!-- Link label text -->
+    <slot />
+  </a>
+
+</template>
+
+
+<script>
+
+  import buttonClassesMixin from './buttonClassesMixin.js';
+
+  /**
+   * kExternalLink creates a styled external link
+   */
+  export default {
+    name: 'kExternalLink',
+    mixins: [buttonClassesMixin],
+    props: {
+      /**
+       * <a href="..."> property
```

Comment:
Update the prop description.

Reply:
can you clarify what you'd like the doc string to be? I thought this was a helpful way of describing the intention but maybe it's not as clear as I thought

Reply:
I think just href or url is clear enough.